src/hooks/core/chat/useMoveChatTypeHook.ts

```diff
@@ -22,7 +23,7 @@ interface MoveToChatChannelParams {
 
 const useMoveChatTypeHook = () => {
   const {localDb, refresh} = useLocalDatabaseHook();
-  const {signedProfileId} = useUserAuthHook();
+  const {signedProfileId, anonProfileId} = useUserAuthHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `anonProfileId` is now being used in addition to the `signedProfileId`. Ensure that this new variable is properly initialized and handled throughout the code to avoid potential null or undefined errors.



<!-- This is an auto-generated comment by OSS CodeRabbit -->